src/rpc/util/hash/ob_hash_sort_utf8mb4.rs

```diff
@@ -3278,6 +3284,61 @@ impl ObHashSortUtf8mb4 {
             if !(is_continuation_byte!(c1)) {
                 return OB_CS_ILSEQ;
             }
+            *wc = ((c & 0x1F) as u32) << 6 | (c1 ^ 0x80) as u32;
+            2
+        } else if c < 0xF0 {
```

Comment:
test it !

Reply:
done ouo
add UT at the end of this file